examples/FluentUI.Demo.Shared/wwwroot/css/site.css

```diff
@@ -343,10 +343,6 @@ code {
         max-width: 100%;
     }
 
-    .navmenu-main-button {
```

Comment:
Removed because it was not visible in the demo on phones, which is exactly where I would expect to see it.